packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -28,155 +39,82 @@ export const Default = {
 };
 
 /**
- * The marker in its 'Cluster' variant in all possible states.
+ * Use the `variant` attribute to change the appearance of the marker.
+ *
+ * - `main`: The default variant.
+ * - `place`: A variant that is used to show a location on the map.
+ * - `cluster`: A variant that is used to show multiple locations on the map.
```

Comment:
Please use "88" for cluster variant. instead of 8

Reply:
Updated